plugin-core/src/main/java/appland/webviews/appMap/AppMapFileEditor.java

```diff
@@ -151,6 +153,38 @@ public void setState(@NotNull FileEditorState state) {
                 }
                 return null;
 
+            case "sidebarSearchFocused":
+                telemetryService.sendEvent("sidebar_search_focused");
+                return null;
+
+            case "clickFilterButton":
+                telemetryService.sendEvent("click_filter_button");
+                return null;
+
+            case "clickTab":
+                if (message == null) return null;
```

Comment:
nit: nesting `if` statements may be a bit easier to read, similar to the handler of `viewSource`. But that's mostly personal choice.

Reply:
because this is in a `switch` and it's always returning `null`, this would things a bit easier to follow.